packages/angular/.eslintrc.json

```diff
@@ -33,6 +33,68 @@
       "rules": {
         "@nx/nx-plugin-checks": "error"
       }
+    },
+    {
+      "files": ["./package.json"],
+      "parser": "jsonc-eslint-parser",
+      "rules": {
+        "@nx/dependency-checks": [
+          "error",
+          {
+            "ignoredDependencies": [
+              "nx",
+              "eslint",
+              "rxjs",
+              "semver",
+              "tslib",
+              "webpack-merge", // This one is different intentionally (from before), not sure why
+              // These are installed by ensurePackage so missing in package.json
+              "@nx/cypress",
+              "@nx/jest",
+              "@nx/rollup",
+              "@nx/storybook",
+              "@nx/vite",
+              "@nx/webpack",
+              "prettier",
+              "typescript",
```

Comment:
I feel that typescript should probably be included. How come they are reported if we include them with `ensurePackage`? I thought the dependency discovery does not track `ensurePackage` calls.